.github/workflows/render_readme.yml

```
@@ -41,8 +41,17 @@ jobs:
     env:
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
     steps:
+      - name: Generate token
+        id: generate_token
+        uses: actions/create-github-app-token@v1
+        with:
+          app_id: ${{ secrets.APP_ID }}
```

actionlint (1.7.9)

avoid using deprecated input "app_id" in action "actions/create-github-app-token@v1": 'app_id' is deprecated and will be removed in a future version. Use 'app-id' instead

(action)